llama/config.py

```diff
@@ -69,12 +69,10 @@ def get_ssm_path(env):
         try:
```

Comment:
This function is now effectively identical to the `get_env()` function above it. I would merge them into a new function called `get_required_env_variable()` that takes the variable name string as an argument rather than leaving them both here.